tests/s3_data_plane_tests.c

```diff
@@ -102,7 +102,11 @@ enum aws_s3_client_tls_usage {
     AWS_S3_TLS_DISABLED,
 };
 
-static int s_test_s3_get_object_helper(struct aws_allocator *allocator, enum aws_s3_client_tls_usage tls_usage) {
+static int s_test_s3_get_object_helper(
+    struct aws_allocator *allocator,
+    enum aws_s3_client_tls_usage tls_usage,
+    struct aws_byte_cursor s3_path,
+    enum AWS_S3_TESTER_SSE_TYPE sse_type) {
```

Comment:
Could we make this enum lower case to match the other enums, and maybe move the argument to be next to the tls_usage?